core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -5511,6 +5507,18 @@ object ZIO extends ZIOCompanionPlatformSpecific with ZIOCompanionVersionSpecific
       }
   }
 
+  object Grafter {
+
+    def apply[E, A](fiberState: Fiber.Runtime[E, A])(implicit trace: Trace): Grafter = {
```

Comment:
Sorry for the nit 😅: AFAICT the type params are unnecessary, you can simply change the param to `Fiber.Runtime[?, ?]`

Reply:
Also let's make this method package-private for now